### PR TITLE
add maldivian holidays

### PR DESCRIPTION
**Pull request by ahmedfaarih**
This PR is to add the [holidays for Maldives.](https://www.timeanddate.com/holidays/maldives/)

### CODE REVIEW

**Comment by Z3d0X** [comment]
Variable dates does not take year into consideration

File: src/Countries/Maldives.php [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            'Republic Day' => '11-11',
+        ], $this->variableHolidays($year));
+    }
+
+    /** @return array<string, CarbonImmutable> */
+    protected function variableHolidays(int $year): array
+    {
+
+        return [
+            'Ramadan Start' => '03-11',
```
Comment (Nielsvanpach) [inline src/Countries/Maldives.php:32]: I don't think the Islamic holidays can be calculated like this

**Comment by Nielsvanpach** [comment]
There are 2 Maldives PRs at the moment. Perhaps you both could work together? https://github.com/spatie/holidays/pull/63

**Comment by Nielsvanpach** [comment]
I'm closing this one in favour of https://github.com/spatie/holidays/pull/63 as this seems more complete. Thanks for your work on this!

**Pull request closed without merging**